package.json

```diff
@@ -38,8 +38,8 @@
       "default": false
     },
     "disableWhenNoConfig": {
-      "title": "Disable when no config file is found",
-      "description": "Either .stylelintrc or stylelint.config.js",
+      "title": "Disable when no config is found",
+      "description": "Either a stylelint property in package.json, a .stylelintrc file or stylelint.config.js file",
```

Comment:
Maybe it would be better to link to here instead? https://stylelint.io/user-guide/configuration/#loading-the-configuration-object

The _actual_ list of supported formats is quite large, and is handled entirely by stylelint.